ipi/engine/barostats.py

```diff
@@ -564,35 +565,31 @@ def pstep(self, level=0):
         dt = self.pdt[
             level
         ]  # this is already set to be half a time step at the specified MTS depth
-        dt2 = dt**2
-        dt3 = dt**3 / 3.0
 
         # computes the pressure associated with the forces at each MTS level.
         press = np.trace(self.stress_mts(level)) / 3.0
         self.p += dt * 3.0 * (self.cell.V * press)
 
         # integerates the kinetic part of the pressure with the force at the inner-most level.
         if level == self.nmtslevels - 1:
+            nbeads = self.beads.nbeads
```

Comment:
accessing members of depend objects has a (small but non-entirely-negligible) cost because of the @property decorator. this was even worse in the 2.6.1 depend implementation, but it still makes sense to avoid too many calls to __getitem__ when the same meber is used multiple times